doc/doc.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -eu
+
+DOC_VERSION=1
+DOC_PATH=/how-to/1
+
+# Used by vuepress
+export DOC_DIR=$DOC_VERSION
+export SITE_BASE=$DOC_PATH/
+
+# Used to specify --no-cache for example
+ARGS=${2:-""}
+
+if [ ! -d "./$DOC_DIR" ]
+then
+  echo "Cannot find $DOC_DIR/. You must run this script from doc/ directory."
+  exit 1
+fi
+
+case $1 in
+  prepare)
+    if [ -d "framework" ];
```

Comment:
We didn't try to updat the repo anymore, instead we do a `rm -rf` of the old framework